### ISSUE TITLE
Colour Picker doesn't work while running NVDA

### ISSUE THREAD

**Issue by afwaisbard** [ThePacielloGroup/CCA-Win#32]
Can't use the Colour Picker while NVDA is running. It results with CCA getting stuck ("Not Responding"). 

OS: Windows 7.

**Comment by patrickhlauke** [comment]
Can't reproduce in Windows 10, so may be a Windows 7 specific issue.

Could you try uninstalling CCA, installing the latest version https://github.com/ThePacielloGroup/CCA-Win/releases/tag/2.4.0.23 (and also updating NVDA with the most recent version, if you've not already) and seeing if the problem still persists?

**Comment by afwaisbard** [comment]
I've tried it on Windows 8.1 as well, and could not reproduce it there either.

I was able to reproduce it on Windows 7 again using the latest CCA and latest NVDA.

The moment I click on the colour picker I see the process memory usage starting to climb at about 10k-12k per second. 

I just want to add something about this issue:
Once I realized that the reason is that I'm running NVDA with CCA, it's not that critical for me. I can always turn NVDA off for a moment while testing contrast with CCA. 
However, while not knowing this is the reason, it can be very frustrating. I even saw a guy at an accessibility conference trying to demo CCA colour picker, and he didn't understand why it got stuck (wish I could help him back then).
Meanwhile I'll do my best sharing this information with my colleagues so they'll be aware there's a workaround and not quit this tool for this.

**Comment by patrickhlauke** [comment]
@afwaisbard I know this is now quite a while ago, but: any chance you could let me know if this is still an issue using the latest released version https://github.com/ThePacielloGroup/CCA-Win/releases/tag/2.5.0 ?

**Comment by burningmetals** [comment]
I just installed the latest version of CCA-Win (2.5.0.0) on Windows 7 Enterprise 64 bit with service pack 1 and running NVDA 2018.1.1. I do not see any issue or conflict in running this combination.

Nice utility!

**Comment by patrickhlauke** [comment]
thanks @burningmetals ... will close this then unless i hear otherwise
